src/services/refactors/extractMethod.ts

```diff
@@ -0,0 +1,885 @@
+/// <reference path="../refactorProvider.ts" />
+/// <reference path="../../compiler/checker.ts" />
+
+/* @internal */
+namespace ts.refactor.extractMethod {

```

Comment:
Why is this `ts.refactor.extractMethod` and not just `ts.refactor`? We aren't using this pattern in _convertFucntionToEs6Class.ts_.